travvy/src/components/Onboarding.vue

```diff
@@ -130,6 +150,53 @@ export default {
             } catch (error) {
                console.log(error)
             }
+      },
+
+      checkForm: function(x){
+              // an array with the current errors and what will be returned to the user
+              this.errors = [];
```

Comment:
We finally have error handling for all input fields!! Out of curiosity, do we want to explicitly indicate whether an input will be optional/"if applicable" (i.e. dream vacation spot or Instagram handles)? Maybe that's something we could quickly add on the frontend.

Reply:
Yeah that is definitely an option, for now I just added 'required' beside the fields which we consider to be mandatory